src/fraqture/photo_countdown.clj

```diff
@@ -0,0 +1,40 @@
+(ns fraqture.photo-countdown
+  (:require [fraqture.drawing]
+            [quil.core :as q]
+            [fraqture.led-array :as led])
+  (:import  [fraqture.drawing Drawing]))
+
+(defn setup [options]
+  {:background-color [0 0 0]
+   :time-left 3
+   :options options})
+
+(defn update-state [state]
+  (-> state
```

Comment:
Since you aren't chaining, it might be better to write this as

`(update-in state [:time-left] dec)`